src/languages/lib/ecmascript.js

```diff
@@ -141,6 +141,7 @@ export const BUILT_IN_VARIABLES = [
   "arguments",
   "this",
   "super",
+  "import",
```

Comment:
We already consider this a `keyword`, is there a reason you think it's a built-in variable?  Typically this is used for "objects"...

Reply:
This addition can be removed since we already highlight it as a keyword.

Reply:
Okay, but wouldn't that make it not have that same color when used as a function, aka `import X from 'y.js'` being colored but `const x = import('y.js')` would just become the default function color?

Reply:
No, that is what `noneOf` resolves... you should be able to test both and see the results.